android/gradle.properties

```diff
@@ -19,4 +19,6 @@
 
 android.useDeprecatedNdk=true
 MYAPP_RELEASE_STORE_FILE=kiwi-rn-hotels.keystore
-MYAPP_RELEASE_KEY_ALIAS=kiwi-rn-hotels
\ No newline at end of file
+MYAPP_RELEASE_KEY_ALIAS=kiwi-rn-hotels
+android.enableAapt2=false
+org.gradle.jvmargs=-Xmx3072m
```

Comment:
See 
https://stackoverflow.com/questions/37090135/to-run-dex-in-process-the-gradle-daemon-needs-a-larger-heap-it-currently-has-a